### PR TITLE
Implement orange hover over for the environment switcher chevron

### PR DESCRIPTION
**Pull request by anthonydillon**
Fixes: https://github.com/juju/juju-gui/issues/1387

### CODE REVIEW

**Comment by anthonydillon** [comment]
# QA
- Hover over the environment switcher and both the chevron and environment name should turn orange

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4524/
Test PASSed.

File: jujugui/static/gui/src/app/components/env-switcher/_env-switcher.scss [hunk 1]
```diff
@@ -14,6 +19,13 @@
         .svg-icon {
             margin-left: 10px;
             vertical-align: text-top;
+            fill: currentColor;
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/env-switcher/_env-switcher.scss]: Nice!

**Comment by huwshimi** [comment]
QA OK :+1:

**Comment by kadams54** [comment]
:+1:

**Comment by anthonydillon** [comment]
Thanks :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**